packages/page-accounts/src/Accounts/Account.tsx

```diff
@@ -355,283 +357,340 @@ function Account ({ account: { address, meta }, className = '', delegation, filt
   ].filter((i) => i),
   [_clearDemocracyLocks, _showOnHardware, _vestingVest, api, delegation, democracyUnlockTx, genesisHash, identity, isDevelopment, isEditable, isExternal, isHardware, isInjected, isMultisig, multiInfos, onSetGenesisHash, proxy, recoveryInfo, t, toggleBackup, toggleDelegate, toggleDerive, toggleForget, toggleIdentityMain, toggleIdentitySub, toggleMultisig, togglePassword, toggleProxyOverview, toggleRecoverAccount, toggleRecoverSetup, toggleUndelegate, vestingVestTx]);
 
+  const handleOnExpandIconClick = useCallback(
+    () => setIsExpanded((isExpanded) => !isExpanded),
+    []
+  );
+
   if (!isVisible) {
     return null;
   }
 
   return (
-    <tr className={className}>
-      <td className='favorite'>
-        <Icon
-          color={isFavorite ? 'orange' : 'gray'}
-          icon='star'
-          onClick={_onFavorite}
-        />
-      </td>
-      <td className='together'>
-        {meta.genesisHash
-          ? <Badge color='transparent' />
-          : isDevelopment
-            ? (
-              <Badge
-                className='devBadge'
-                color='orange'
-                hover={t<string>('This is a development account derived from the known development seed. Do not use for any funds on a non-development network.')}
-                icon='wrench'
-              />
-            )
-            : (
-              <Badge
-                color='orange'
-                hover={
+    <>
```

Comment:
Dear reviewer - because of this change, all the code of a TR got indented, so it is shows as changed. Be notified as you may want to optimize diff showing for this PR in github (Split, no whitespaces)